apps/framework-docs/src/pages/moose/streaming/create-stream.mdx

```diff
@@ -109,20 +109,20 @@ export const rawIngestApi = new IngestApi<RawData>("raw_data", {
 
 <Python>
 ```py filename="StreamObject.py" copy {8-12}
-from moose_lib import IngestPipeline, Key
+from moose_lib import Stream, StreamConfig, Key, IngestApi, IngestConfig
 from pydantic import BaseModel
 
 class RawData(BaseModel):
     id: Key[str]
     value: int
 
 raw_table = OlapTable[RawData]("raw_data")
-raw_stream = Stream[RawData]("raw_data", {
+raw_stream = Stream[RawData]("raw_data", StreamConfig(
   destination: raw_table # Optional: Specify a destination table for the stream, sets up a process to sync data from the stream to the table
-})
-raw_ingest_api = IngestApi[RawData]("raw_data", {
+))
+raw_ingest_api = IngestApi[RawData]("raw_data", IngestConfig(
```

Comment:
this only works for me with "IngestConfigWithDestination". not sure if that is a bug, or intended?

Reply:
also homepage docs says "IngestAPIConfig" which does NOT work. we really need to test docs code better.

Reply:
cc @phiSgr on the potential bug

Reply:
seems like a bug

Reply:
fixed homepage